packages/stacked_cli/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 - Refactors `_handleVersion` to use PubService
 - Updates `_handleFirstRun` for prettier notification
 - Shows notification when new version is available
+- Adds support for `version` custom dimension
+- Adds support for `name` custom dimension
+- Sends `stacked_cli` version custom dimension on every event
```

Comment:
Can we move this into a new section for 1.2.0 and update the version to 1.2.0 please